charts/karpenter/values.yaml

```diff
@@ -51,7 +51,10 @@ podDisruptionBudget:
   maxUnavailable: 1
 # -- SecurityContext for the pod.
 podSecurityContext:
+  runAsNonRoot: true
   fsGroup: 65532
+  seccompProfile:
```

Comment:
Similar question about the seccompProfile

Reply:
The same as above. In addition, by providing a default it makes the intent clear and supports policies that don't consider the implicit defaults.

Reply:
See comment above